src/common/attributes/attributes/attribute-balance.component.ts

```diff
@@ -45,10 +46,10 @@ export class AttributeBalanceComponent implements OnChanges {
     }
 
     if (changes['tokenDecimals'] || changes['attribute'] || changes['decimals']) {
-      let converted: number | null;
+      let converted: BigNumber | null;
 
       try {
-        converted = Math.max(0, parseInt(this.attribute.value as string, 10)) / Math.pow(10, this.decimals);
+        converted = BigNumber.maximum(0, (new BigNumber(this.attribute.value))).dividedBy(new BigNumber(Math.pow(10, this.decimals)));
```

Comment:
I removed the `parseInt`. I think it's ok bc that's only needed for the response from RPC. 
Lmk if you think it will be an issue

Reply:
> I think it's ok bc that's only needed for the response from RPC

No, I think that's the because the input param is `this.attribute.value as string` which is a string, but in this case, if I understand correctly, BigNumber can expect this.attribute.value as input param without specifying its type.

Reply:
@yurikoinaba Did we check that `this.decimals` is consistent accross all the pages?
I mean if we take one page that rendered correctly and another that didn't, is `this.decimals` the same value for both?

Reply:
@rdoria1  Yes, `this.decimals` 18 and is consistent.

The problem is `parseInt('1e+21', 10) = 1` which makes `Math.max(0, 1) = 1`, and so the value is always showing 10^-18

Reply:
I misread your comment. Let me check

Reply:
@rdoria1 I checked `this.decimals` in this component on `/transfer` and `/account` pages
https://github.com/HorizenLabs/zkverify-explorer-ui/blob/32e0459180f2419ac9f693907844c410cdd2789c/src/common/balance/balance.common.component.ts#L65

It is always 18. 
It looks like most other places are using this balance component
cc @santigamo